snap/hooks/configure

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exit 0
```

Comment:
I'm not familiar with snap hooks, but I'm guessing this empty script is necessary in order for `snap set` to work?

Reply:
Correct, it might be an idea to check to see if either memprof or cpuprof is set and restart the snap within the configure hook, what are your thoughts on that?

Reply:
An unconditional restart on `snap set` wouldn't be bad -- if this hook only gets run on snap set (or unset?).

Reply:
hmm yes, on reflection I don't know if there is a way inside a hook to check that that you are only changing cpu/mem prof from nil to true. You can obviously use snap get to see what these config items are set to but it wouldn't stop the snap from restarting when other options are set, for example, `snap set fooOption` whilst cpuprof is true.

I think this is ready to land having then.